extensions/systemd/index.js

```diff
@@ -22,6 +22,9 @@ class SystemdExtension extends Extension {
                 }
 
                 return false;
+            },
+            userSkip: ({instance}) => {
```

Comment:
I think this approach makes sense - though I wonder if it might be better named, like `onSkip` or something 🤔

Reply:
My issue with `onSkip` is it's ambiguous as to what skipped (auto or user) - thoughts on `onUserSkip`?

Reply:
yeah that sounds good 👍